### DISCUSSION TITLE
/packages default project path

### DISCUSSION THREAD

**Discussion by dillingham** [laravel/framework#37312] [Ideas]
Would be 🔥 if laravel/laravel had a `/packages` folder by default and composer.json
```json
"repositories": [
  {
      "type": "path",
      "url": "./packages/*"
  },
]
```

Maybe even  a `php artisan eject vendor/package` which adds it to /packages and updates

**Comment by Wulfheart** [comment]
And what would the benefits be?

**Reply by dillingham** [reply]
Basically just a standard plugins directory for manually adding packages locally.
Nova uses /nova-components for this purpose. If laravel had a /packages or /plugins folder by default, would of taken advantage of that I assume as would others in times of need
